### PR TITLE
{phys}[intel/2017a] Siesta v4.0, Siesta v4.1-b2

### PR DESCRIPTION
**Pull request by migueldiascosta**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
nscc04 - Linux RHEL 6.8, Intel(R) Xeon(R) CPU E5-2690 v3 @ 2.60GHz, Python 2.7.13
See https://gist.github.com/2291e27c3c265bf74a057875da7d7530 for a full test report.

**Comment by akesandgren** [comment]
These can be more generic.

vl = version.split('-')
source_urls = ['https://launchpad.net/siesta/%s/%%(version)s/+download' % vl[0]]
is better i think.

And a checksum to go with that perhaps?

File: easybuild/easyconfigs/s/Siesta/Siesta-4.0-intel-2017a.eb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+toolchainopts = {'usempi': True}
+
+sources = [SOURCELOWER_TGZ]
+vl = version.split('-')
+source_urls = ['https://launchpad.net/siesta/%s/%%(version)s/+download' % vl[0]]
+checksums = ['c4aa5f4cf2eabefaaacd3ef6350ec8fd']
+
+dependencies = [('netCDF-Fortran', '4.4.4')]
+
+parallel = 1
```
Comment (boegel) [inline easybuild/easyconfigs/s/Siesta/Siesta-4.0-intel-2017a.eb]: @migueldiascosta This is no longer needed, right? It's handled by the easyblock?

File: easybuild/easyconfigs/s/Siesta/Siesta-4.1-b2-intel-2017a.eb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+toolchainopts = {'usempi': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+vl = version.split('-')
+source_urls = ['https://launchpad.net/siesta/%s/%%(version)s/+download' % vl[0]]
+checksums = ['6dd700aa79b0da39bbd3233c220d630f']
+
+dependencies = [('netCDF-Fortran', '4.4.4')]
+
+parallel = 1
```
Comment (boegel) [inline easybuild/easyconfigs/s/Siesta/Siesta-4.1-b2-intel-2017a.eb]: versions newer than `4.1` support building in parallel (according to the easyblock)?

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
nscc04 - Linux RHEL 6.8, Intel(R) Xeon(R) CPU E5-2690 v3 @ 2.60GHz, Python 2.7.13
See https://gist.github.com/0b0fe5b40b7456cd3fbb985d02cfce33 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2053.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/90c34cf0067f33785273d4aa99e565db for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2482.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/cea95a6c7af2e7b7cd6030d52b1ceb02 for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Going in, thanks @migueldiascosta!

**Pull request merged**